src/RhythmRunner.cpp

```diff
@@ -108,8 +108,10 @@ int main(int argc, char** argv) {
   std::shared_ptr<LevelGenerator> levelGenerator =
       std::make_shared<LevelGenerator>(music);
 
-  std::shared_ptr<Level> level = std::make_shared<Level>(
-      levelGenerator->getMusic(), levelGenerator->generateLevel());
+  std::shared_ptr<Level> level = levelGenerator->generateLevel();
+
+  //std::shared_ptr<Level> level = std::make_shared<Level>(
+  //    levelGenerator->getMusic(), levelGenerator->generateLevel());
```

Comment:
The level generator makes more sense to return a full level including the collectibles platforms etc instead of just platforms.